src/app/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+
+interface ToastProps {
+  message: string;
+  linkText?: string;
+  linkUrl?: string;
+  closeText?: string;
+  onClose: () => void;
+}
+
+export const Toast: React.FC<ToastProps> = ({
+  message,
+  linkText,
+  linkUrl,
+  closeText,
+  onClose,
+}) => {
+  const [mounted, setMounted] = useState(false);
+
+  // useEffect only runs on the client, so now we can safely show the UI
+  useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  if (!mounted) return null;
+
+  return (
+    <div className="toast toast-end z-10">
+      <div className="alert max-w-[20rem] flex flex-col">
```

Comment:
im thinking if it would be better to make it full width when its on mobile view

Reply:
Done